translations/ja.json

```
@@ -864,55 +864,55 @@
-	"error stopping tunnel": "tunnel を停止する際にエラーが発生しました",
-	"error: --output must be 'text', 'yaml' or 'json'": "",
-	"error: --output must be 'yaml' or 'json'": "エラーです。 --output は「 yaml 」、あるいは「 json 」である必要があります",
-	"experimental": "",
-	"failed to add node": "",
-	"failed to open browser: {{.error}}": "ブラウザを起動するのに失敗しました。 {{.error}}",
-	"failed to save config": "",
-	"failed to set cloud shell kubelet config options": "",
-	"failed to start node": "",
-	"fish completion failed": "",
-	"fish completion.": "",
-	"if true, will embed the certs in kubeconfig.": "有効であれば、Kubernetes の設定ファイルに証明書を埋め込みます",
-	"if you want to create a profile you can by this command: minikube start -p {{.profile_name}}": "minikube のプロフィールを作成する場合は、以下のコマンドで作成できます。 minikube start -p {{.profile_name}}",
-	"initialization failed, will try again: {{.error}}": "初期化が失敗しました。再施行します。 {{.error}}",
-	"invalid kubernetes version": "",
-	"keep the kube-context active after cluster is stopped. Defaults to false.": "",
-	"kubeadm detected a TCP port conflict with another process: probably another local Kubernetes installation. Run lsof -p\u003cport\u003e to find the process and kill it": "kubeadm が他のプロセス (おそらくインストール済みの他のローカル Kubernetes) との TCP ポート競合を検出しました。 lsof -p\u003cport\u003e を実行して、そのプロセスを kill してください",
-	"kubectl and minikube configuration will be stored in {{.home_folder}}": "kubectl と minikube の設定は {{.home_folder}} に保存されます",
-	"kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'": "kubectl が見つかりません。kubectl が必要であれば、'minikube kubectl -- get pods -A' を試してください。",
-	"kubectl proxy": "kubectl proxy",
+	"disable failed": "無効化に失敗しました",
+	"dry-run mode. Validates configuration, but does not mutate system state": "dry-run モード。設定は検証しますが、システムの状態は変更しません",
+	"dry-run validation complete!": "dry-run の検証が終了しました!",
+	"enable failed": "有効化に失敗しました",
+	"error creating clientset": "clientset 作成中にエラー",
+	"error getting primary control plane": "最初のコントロールプレーン取得中にエラー",
+	"error getting ssh port": "SSH ポートを取得中にエラー",
+	"error initializing tracing: {{.Error}}": "トレーシング初期化中にエラー: {{.Error}}",
+	"error parsing the input ip address for mount": "マウント用に入力された IP アドレスをパース中にエラー",
+	"error provisioning guest": "ゲストのプロビジョン中にエラー",
+	"error starting tunnel": "トンネル開始中にエラー",
+	"error stopping tunnel": "トンネル停止中にエラー",
+	"error: --output must be 'text', 'yaml' or 'json'": "エラー: --output は 'text'、'yaml'、'json' のいずれかでなければなりません",
+	"error: --output must be 'yaml' or 'json'": "エラー: --output は 'yaml'、'json' のいずれかでなければなりません",
+	"experimental": "実験的",
+	"failed to add node": "ノード追加に失敗しました",
+	"failed to open browser: {{.error}}": "ブラウザー起動に失敗しました: {{.error}}",
+	"failed to save config": "設定保存に失敗しました",
+	"failed to set cloud shell kubelet config options": "クラウドシェル kubelete 設定オプションの設定に失敗しました",
+	"failed to start node": "ノード開始に失敗しました",
+	"fish completion failed": "fish のコマンド補完に失敗しました",
+	"fish completion.": "fish のコマンド補完です。",
+	"if true, will embed the certs in kubeconfig.": "true の場合、Kubernetes に証明書を埋め込みます。",
```

```suggestion
	"if true, will embed the certs in kubeconfig.": "true の場合、kubeconfig に証明書を埋め込みます。",
```
Original word is "kubeconfig".